### PR TITLE
[iOS] Add Speaker View

### PR DESCRIPTION
**Pull request by yanamura**
## Issue
- wip #491

### TODO
- show session

## Overview (Required)
- just show speaker info
- I'll do session info on the other PR

## Links
-

### CODE REVIEW

**Comment by ry-itto** [comment]
@yanamura 
Is this finished? If not, please make this PR `Draft` or add [WIP] at the beginning of this PR's title. 🙏

**Comment by yanamura** [comment]
@ry-itto I want to split PR. Because this PR includes fix Navigation bar changes, this feature is required from other issues.

File: ios-base/DroidKaigi 2020/AppDelegate.swift [hunk 1]
```diff
@@ -8,8 +8,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if #available(iOS 13, *) {
         } else {
             let window = UIWindow(frame: UIScreen.main.bounds)
-            UINavigationBar.appearance().tintColor = ApplicationScheme.shared.colorScheme.onPrimaryColor
-            UINavigationBar.appearance().backgroundColor = ApplicationScheme.shared.colorScheme.primaryColor
+
+            let backButtonBackgroundImage = #imageLiteral(resourceName: "ic_back")
```
Comment (ry-itto) [inline ios-base/DroidKaigi 2020/AppDelegate.swift]: 👍

File: ios-base/DroidKaigi 2020/Session/Views/SpeakerViewController.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import Foundation
+import ios_combined
+import Nuke
+import UIKit
+
+final class SpeakerViewController: UIViewController {
```
Comment (ry-itto) [inline ios-base/DroidKaigi 2020/Session/Views/SpeakerViewController.swift]: Please move this file to new directory. Such as `$SRCROOT/Droid Kaigi/Speaker/Views`.

File: ios-base/DroidKaigi 2020/Session/Views/SpeakerViewController.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        navigationController?.navigationBar.tintColor = .black
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+    }
```
Comment (ry-itto) [inline ios-base/DroidKaigi 2020/Session/Views/SpeakerViewController.swift]: This is unnecessary. Please remove this 🙏

**Comment by ry-itto** [comment]
@yanamura 
The way to apply `color` for `navigationBar` is changed by https://github.com/DroidKaigi/conference-app-2020/pull/620 . So this PR has some conflicts. Please resolve it 🙏

File: ios-base/DroidKaigi 2020/Session/Views/SpeakerViewController.swift (hunk 2 above)
Comment (yanamura) [reply ios-base/DroidKaigi 2020/Session/Views/SpeakerViewController.swift]: oh sorry, i misunderstood project structures

**Comment by yanamura** [comment]
fixed

File: ios-base/DroidKaigi 2020/AppDelegate.swift [hunk 4]
```diff
@@ -9,7 +9,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if #available(iOS 13, *) {
         } else {
             let window = UIWindow(frame: UIScreen.main.bounds)
-            UINavigationBar.appearance().isTranslucent = false
```
Comment (ry-itto) [inline ios-base/DroidKaigi 2020/AppDelegate.swift]: Please add this 🙏

Comment (yanamura) [reply ios-base/DroidKaigi 2020/AppDelegate.swift]: 🙇

**Pull request merged**